src/analysis/complex/upper_half_plane/functions_bounded_at_infty.lean

```diff
@@ -50,8 +50,7 @@ def is_zero_at_im_infty {α : Type*} [has_zero α] [topological_space α] (f : 
 zero_at_filter at_im_infty f
 
 lemma zero_form_is_bounded_at_im_infty {α : Type*} [normed_field α] :
-  is_bounded_at_im_infty (0 : ℍ → α) :=
-zero_is_bounded_at_filter at_im_infty
+  is_bounded_at_im_infty (0 : ℍ → α) := zero_is_bounded_at_filter _
```

Comment:
I don't see the point of this change. Besides this, LGTM.

Reply:
Oh sorry that was me reverting the change incorrectly! its back to the original now :)